crates/subspace-farmer/src/cluster/plotter.rs

```diff
@@ -410,6 +410,7 @@ impl ClusterPlotter {
                                     break 'outer;
                                 }
                                 ResponseProcessingResult::Continue => {
+                                    trace!("Continue");
```

Comment:
Please, add some identifiers (Maybe `free_instance`?) to the log here and in other parts of the `match` as well as more clear message ("Plotting response result: abort"). It lacks the context now.

Reply:
It is already inspected with farm and sector and there are other messages that are happening right before this, I just need to know the result of `process_response_notification` function call or rather I was curious if it returned anything or hanged in there.